File: Examples/Examples/Chart/Documentation Snippets/ChartDocsSplineAreaSeries.cs
using System;
using TelerikUI;
using Foundation;
using System.Collections.Generic;

namespace Examples
{
	public class ChartDocsSplineAreaSeries
	{
		TKChart chart = new TKChart();

		void snippet1 ()
		{
			// >> chart-spline-area-cs
			var categories = new [] { "Greetings", "Perfecto", "NearBy", "Family Store", "Fresh & Green" };
			var values = new [] { 70, 75, 58, 59, 88 };
			var pointsWithCategoriesAndValues = new List<TKChartDataPoint> ();
			for (int i = 0; i < categories.Length; ++i) {
				pointsWithCategoriesAndValues.Add (new TKChartDataPoint (new NSString (categories [i]), new NSNumber (values [i])));
			}
			chart.AddSeries (new TKChartSplineAreaSeries (pointsWithCategoriesAndValues.ToArray ()));
			// << chart-spline-area-cs
		}
	}
}

File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/ChartDocsSplineAreaSeries.swift
//
//  ChartDocsSplineAreaSeries.swift
//  TelerikUIExamplesInSwift
//
//  Copyright © 2016 Telerik. All rights reserved.
//

class ChartDocsSplineAreaSeries: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        let chart = TKChart(frame: CGRectInset(self.view.bounds, 10, 10))
        self.view.addSubview(chart)
        
        // >> chart-spline-area-swift
        let categories = ["Greetings", "Perfecto", "NearBy", "Family Store", "Fresh & Green" ];
        let values = [70, 75, 58, 59, 88]
        var pointsWithCategoriesAndValues = [TKChartDataPoint]()
        for i in 0 ..< categories.count {
            pointsWithCategoriesAndValues.append(TKChartDataPoint(x: categories[i], y: values[i]))
        }
        
        let series = TKChartSplineAreaSeries(items: pointsWithCategoriesAndValues)
        chart.addSeries(series)
        // << chart-spline-area-swift
    }
}
